README.md

```diff
@@ -443,6 +444,15 @@ Metrics/CyclomaticComplexity:
   Severity: warning
 ```
 
+## Details 
+
+Individual cops can be embellished with extra details in offense messages:
+
+```yaml
+Metrics/LineLength:
```

Comment:
You can format the message like this and it will still print the same way:
```yaml
Metrics/LineLength:
  Details: >-
    If lines are too short, text becomes hard to read because you must
    constantly jump from one line to the next while reading. If lines are too
    long, the line jumping becomes too hard because you "lose the line" while
    going back to the start of the next line.  80 characters is a good
    compromise.
```